kolibri/core/logger/api.py

```diff
@@ -1086,6 +1086,7 @@ class GenerateCSVLogRequestViewSet(viewsets.ModelViewSet):
     """
 
     permission_classes = (KolibriAuthPermissions,)
+    filter_backends = (KolibriAuthPermissionsFilter,)
```

Comment:
I'm thinking that you also need `DjangoFilterBackend` here since a `filter_class` is in use. Perhaps some unit tests would be good :)